rationale.md

```diff
@@ -6,7 +6,17 @@ up when you explore the SDK.
 
 ## Versioning and Releasing
 
-The OpenTelemetry Applications and OpenTelemetry Spec itself use semver v2.
+This document describes the versioning and stability policy of components
+shipped from this repository, as per the [OpenTelemetry versioning and stability
+specification](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/versioning-and-stability.md).
+
+The OpenTelemetry Applications, the OpenTelemetry Spec itself and this repo follows [SemVer V2](https://semver.org/spec/v2.0.0.html) guidelines.
+This means that, for any stable packages released from this repo, all public APIs will remain [backward
```

Comment:
Is it worth explaining semver in detail? I think the link to semver is generally sufficient, and many are familiar with the scheme.

Reply:
I don't think it would hurt? Pulling out the "breaking changes" part because that's the part of semver that customers care about instead of having to go through the entire semver spec.

Reply:
SemVer is quite stable, so it probably doesn't matter much either way. I suppose it's just a nitpick of mine to duplicate information when a hyperlink will do :)